src/main/java/lab3/ui/Console.java

```diff
@@ -0,0 +1,31 @@
+package lab3.ui;
+import java.util.Scanner;
+import lab3.game.Game;
+public class Console {
+
+    // TODO: create a CLASS method to display a tictactoe board
+
+    // TODO: create a CLASS method to get the next move from a player
+    private Game game;
+    private Scanner scanner;
+
+    public Console() {
+        this.game = new Game();
+        this.scanner = new Scanner(System.in);
+    }
+
```

Comment:
The intent was for this class to simply contain a number of class methods that help display things on the console or obtain user input.  Having it keep a Game object, and putting the startGame method in here puts game logic in here rather than in your Game class where it should be.